unfetter-discover/unfetter-ui
# Issue 995/assess3 module

fixes unfetter-discover/unfetter#995

# problem
assessments 2.0 needs to begin replacing its current sensors with the capability baselines from assessments 3.0

# changes
copied the assessments 2.0 into two directories, a v2 and v3.  The v3 is labeled as assessments beta in the info bar and header and app menu.  Both should operate the same at the moment.

# test
* Note, when testing, ensure assessments 2 and assessment 3 links do not cross over to each other
* click the assessments 2.0, verify all the features work
* click the assessments 3.0, verify all the features work
